docs/docs-beta/sidebars.ts

```diff
@@ -1,4 +1,4 @@
-import type {SidebarsConfig} from '@docusaurus/plugin-content-docs';
+import type { SidebarsConfig } from '@docusaurus/plugin-content-docs';
```

Comment:
i feel like every other commit either adds or removes spaces from {these} { things } 
@salazarm do u know why

Reply:
Hmm, are these settings not configured in .eslintrc? I'm guessing it's different local configs on everyone's machine and no checked in config that enforces a specific rule?